.github/workflows/push-training-ecs.yml

```diff
@@ -32,14 +32,14 @@ on:
 
 env:
   AWS_REGION: "us-east-1" # set this to your preferred AWS region, e.g. us-west-1
-  ECR_REPOSITORY: "dlp-backend-image" # set this to your Amazon ECR repository name
```

Comment:
will this ECR repo be removed since it won't be used?

Reply:
i just renamed the repository to `training`, the diff should show it if you view the entire file

Reply:
i see. im wondering if terraform would remove the old ecs service due to the rename?

Reply:
deleted the old ecr repository

Reply:
thx